### ISSUE TITLE
Doesn't work with new Gmail

### ISSUE THREAD

**Issue by laike9m** [adam-p/markdown-here#504]
I'm not sure if it's just me or a bug, but markdown here doesn't work in new Gmail. By "doesn't work" I mean when I click the "Markdown toggle" button nothing happened. I found that Chrome store's version is older than GitHub, so I downloaded source code from GitHub and installed again, still didn't work.

**Comment by sh78** [comment]
Same here. Using the new material design Gmail and the latest stable Google Chrome. Clicking the app icon or right-clicking and using the context menu does nothing. I checked the JavaScript console, and there are no errors on click. I'm on macOS High Sierra.

**Comment by sh78** [comment]
Yes, but no restart required back when I made that comment.

**Comment by filmil** [comment]
@sh78 Yeah, I tried it yesterday, but no dice.  Markdown here does not work on latest gmail, any way you slice it.

**Comment by sh78** [comment]
It's still working for me here. I'm on the latest version of Chrome and
Markdown Here from the Chrome Web Store.

[image: Screen Shot 2018-08-07 at 11.38.15.png]
[image: Screen Shot 2018-08-07 at 11.38.31.png]

**Comment by laike9m** [comment]
@sh78 Still not working for me, no matter how many times I tried. BTW I was installing on a new computer this time.

**Comment by sh78** [comment]
Hmm. I'm on a 2018 MacBook Pro, macOS Sierra latest. Maybe it's an OS
discrepancy?

**Comment by trentks** [comment]
It functions for me in the new gmail, but the "forgot-to-render" check is dysfunctional in that it won't prompt you to render the markdown when hitting send when needed. 

Markdown 2.13.1 with Firefox 61.0.2 (64-bit), on Manjaro Linux.

**Comment by error0x404** [comment]
It works for me at least, it seems like it's just a re-installation/restart issue.

**Comment by nmcbride** [comment]
I just found out about Markdown Here and went to try it in the new Gmail and it didn't not work for me. Nothing happened at all.

**Comment by djnorrisdev** [comment]
Not working for me either.

**Comment by aminukano585** [comment]
Thanks, @djnorrisdev machine restart works for me as well!

**Comment by filmil** [comment]
Any plans to investigate and fix this issue?
No amount of machine restarts fix the issue for me.  It's been a very long time.

**Comment by izhangshiying** [comment]
@laike9m 
Here is one kind of solution
Mine
- Chrome
- Mac

### Reset Site access
- I try to Reset Site access in extensions 
- Choose `on all sites ` to ` On click `
- then it's ok to move on.

![Screen Shot 2019-08-01 at 4 33 12 PM](https://user-images.githubusercontent.com/19412465/62278391-9298ad80-b47a-11e9-9a11-cfd0d58d3591.png)
![Screen Shot 2019-08-01 at 4 33 16 PM](https://user-images.githubusercontent.com/19412465/62278398-95939e00-b47a-11e9-8dab-894158da8500.png)
![Screen Shot 2019-08-01 at 4 33 28 PM](https://user-images.githubusercontent.com/19412465/62278400-975d6180-b47a-11e9-8111-2a194cec7a4d.png)

**Comment by ElectricRCAircraftGuy** [comment]
This issue can be closed. I just installed Markdown Here from the Chrome store minutes ago and used it for the first time ever and it works in the latest gmail. 

HOWEVER, IT DID *NOT* WORK IMMEDIATELY AFTER INSTALLING THE CHROME EXTENSION! I had to first restart chrome by typing `chrome://restart` into the search bar. Then, **after Chrome restarted**, it worked fine in Gmail. Solved!

**Comment by ElectricRCAircraftGuy** [comment]
(Doesn't work in the new Google Sites though): https://github.com/adam-p/markdown-here/issues/574

**Comment by DanielGoldfarb** [comment]
I'm not sure if this is the same thing that I am seeing.

- I write a gmail with markdown.  
- Right click and choose markdown toggle.  
- Everything looks great
- On the receiver's side, no markdown, no html, just plain text :-(

Can anyone help please?
